test/src/main/scala/scala/test/twitter_scrooge/bare_jar_test/BUILD

```diff
@@ -0,0 +1,21 @@
+load("//scala:scala.bzl", "scala_binary")
+load("//thrift:thrift.bzl", "thrift_library")
+load("//twitter_scrooge:twitter_scrooge.bzl", "scrooge_scala_library")
+
+java_import(
```

Comment:
this, in my opinion, should be `filegroup`: https://docs.bazel.build/versions/master/be/general.html#filegroup

this is not a java library (since it likely has no java bytecode inside it). If it does, we handle it with `remote_jar` in scrooge already.